ci/jobs/pr-docs.yaml

```diff
@@ -0,0 +1,187 @@
+- job:
+    name: 'docs-pulp-pr'
+    defaults: ci-workflow-runtest
+    project-type: matrix
+    node: 'master'
+    axes:
+      - axis:
+          type: label-expression
+          name: node-type
+          values:
+            - f25-np
+    parameters:
+        - string:
+            name: sha1
+            default: ''
+            description: |
+                The hash or refname of the branch to build: for example: origin/pr/9/head
+        - matrix-combinations:
+            name: MATRIX_AXIS
+    properties:
+      - dev-ownership
+      - github:
+          url: https://github.com/pulp/pulp/
+    scm:
+        - git:
+            url: 'https://github.com/pulp/pulp.git'
+            refspec: '+refs/pull/*:refs/remotes/origin/pr/*'
+            branches:
+              - "${sha1}"
+            basedir: pulp
+            skip-tag: True
+            wipe-workspace: false
+    triggers:
+      - github-pull-request:
+          admin-list:
+            - pulpbot
+          org-list:
+            - pulp
+          # Poll once per minute since our jenkins isn't public and web hooks aren't an option
+          cron: '* * * * *'
+          status-context: 'Docs Builder'
+          status-url: 'https://pulpadmin.fedorapeople.org/jenkins/jobs/${JOB_NAME}/builds/${BUILD_ID}/'
+          trigger-phrase: 'ok test'
+          allow-whitelist-orgs-as-admins: true
+          # only-trigger-phrase: true
+          white-list-target-branches:
+            - 2.13-dev
+            - 2.13-release
+            - master
```

Comment:
The doc builder should also work with the 3.0-dev branch after https://pulp.plan.io/issues/2819 is done